sections/sprint-4/networking-updates.tex

```diff
@@ -39,6 +39,18 @@ \subsubsection{Introducing TCP}
 This would remove the need for clients sending UDP acknowledgments for game crucial data.
 Because of the reliability, and needing established connections, TCP messages will be slower to transmit.
 As one of the focuses for the project is creating a networking solution capable of continuously updating positional data as quickly as possible, this is not an ideal solution for transmitting the data when the game is being played.
+\begin{table}[h!]
+    \begin{tabular}{|l|l|l|l|}
+        \hline
+            & Plus                                                                                                            & Minus                                                                                                                         & Interesting                                                                                                                                                             \\ \hline
+        TCP & \begin{tabular}[c]{@{}l@{}}Reliable\\ Built in acknowledgements\\ Two way communication\end{tabular}            & \begin{tabular}[c]{@{}l@{}}Slower than UDP\\ More overhead because\\  of reliability and\\ two way communication\end{tabular} & \begin{tabular}[c]{@{}l@{}}For critical messages that concern \\ the state of the game TCP can \\ confirm if all players have received \\ the update.\end{tabular}      \\ \hline
+        UDP & \begin{tabular}[c]{@{}l@{}}Faster than TCP\\ Less overhead because it's a \\ one way communication\end{tabular} & \begin{tabular}[c]{@{}l@{}}Not as reliable\\ No built in \\ aknowledgements\end{tabular}                                      & \begin{tabular}[c]{@{}l@{}}Because UDP doesn't check if \\ the clients have recieved the \\ correct packages it can send \\ updates a lot faster than TCP.\end{tabular} \\ \hline
```

Comment:
```suggestion
        UDP & \begin{tabular}[c]{@{}l@{}}Faster than TCP\\ Less overhead because it is a \\ one way communication\end{tabular} & \begin{tabular}[c]{@{}l@{}}Not as reliable\\ No built in \\ aknowledgements\end{tabular}                                      & \begin{tabular}[c]{@{}l@{}}Because UDP does notcheck if \\ the clients have recieved the \\ correct packages it can send \\ updates a lot faster than TCP.\end{tabular} \\ \hline
```